### PR TITLE
Added Basic Victory Line Chart with Brush and Zoom

### PR DESCRIPTION
**Pull request by craigericlewis**

### CODE REVIEW

**Comment by craigericlewis** [comment]
image

File: react-d3-chart/src/components/Victory/Context/index.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    () =>
+      contextFilteredData.map((lineData, index) => {
+        return (
+          <VictoryLine
+            animate={false}
+            interpolation="linear"
+            key={`line${index + 1}`}
+            data={lineData}
+            style={{
+              data: { stroke: colour(index.toString()) },
```
Comment (zhm12359) [inline react-d3-chart/src/components/Victory/Context/index.tsx:63]: nit: color ? prefer US spelling...

File: react-d3-chart/src/components/Victory/LineChart/index.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+) =>
+  data.map((line) => {
+    return line.filter(
+      (d) => d.x >= selectedDomain.x[0] && d.x <= selectedDomain.x[1]
+    );
+  });
+
+const downSample = (filtered: Coordinate[][], maxPoints: number) => {
+  if (filtered.length > 0 && filtered[0].length > maxPoints) {
+    const k = Math.ceil(filtered[0].length / maxPoints);
```
Comment (zhm12359) [inline react-d3-chart/src/components/Victory/LineChart/index.tsx:59]: duplicate code for context line. make it into an util function?

File: react-d3-chart/src/components/Victory/LineChart/VictoryLineChart.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+ranges.split(',').forEach((range) => {
+  const l = parseInt(range.trim().split('-')[0]);
+  const r = parseInt(range.trim().split('-')[1]) + 1;
+
+  for (let i = l; i < r; i++) {
+    const temp = graphData.trace[i].c3.map((value, index) => ({
+      x: index,
+      y: value,
+    }));
+    data.push([...temp]);
```
Comment (zhm12359) [inline react-d3-chart/src/components/Victory/LineChart/VictoryLineChart.stories.tsx:19]: data is an array of arrays? In my mind it should be [{x:1, y:2}, {x:2, y:3}, ...]

**Pull request merged**